src/Nest/Mapping/Types/Specialized/TokenCount/TokenCountAttribute.cs

```diff
@@ -14,6 +14,12 @@ public string Analyzer
 			set => Self.Analyzer = value;
 		}
 
+		public bool EnablePositionIncrements
+		{
+			get => Self.EnablePositionIncrements.GetValueOrDefault();
```

Comment:
```suggestion
			get => Self.EnablePositionIncrements.GetValueOrDefault(true);
```

The default is `true`: https://www.elastic.co/guide/en/elasticsearch/reference/current/token-count.html#token-count-params